lib/platform/bitbucket-server/utils.ts

```diff
@@ -32,8 +31,8 @@ export function prInfo(pr: BbsRestPr): BbsPr {
 }
 
 const addMaxLength = (inputUrl: string, limit = 100): string => {
-  const { search, ...parsedUrl } = url.parse(inputUrl, true); // eslint-disable-line @typescript-eslint/no-unused-vars
-  const maxedUrl = url.format({
+  const { search, ...parsedUrl } = parseUrlLegacy(inputUrl, true); // eslint-disable-line @typescript-eslint/no-unused-vars
+  const maxedUrl = formatUrl({
```

Comment:
This destructuring fails when `search` is null. Can you set a default value, or not destructure, and check the value afterwards instead?

Reply:
This will work as before. `parseUrlLegacy` and `formatUrl` are simply reexports of the `url` module, so no behavior change here.